python/examples/simple/requirements.txt

```diff
@@ -1,2 +1,2 @@
 six==1.10.0
-metaparticle_pkg==0.4.1
+metaparticle_pkg==0.4.1
```

Comment:
Not a big fan of explicit package #s in master, but my opinions are better expressed in https://github.com/metaparticle-io/package/pull/126 – YMMV

Reply:
Actually, this would easily get around a few of the problems i was investigating... I'll make the change. (One outstanding problem with this approach: this wont make it any easier to dev on this package and test it within a container though)